django/applications/catmaid/control/treenodeexport.py

```diff
@@ -173,7 +173,7 @@ def export_single_node(self, treenode) -> None:
             # Save image in output path, named <treenode-id>.tiff
             image_name = "%s.tiff" % treenode.id
             treenode_image_path = os.path.join(output_path, image_name)
-            img.write(treenode_image_path)
+            img.save(treenode_image_path)
```

Comment:
This I didn't test, but I would assume it also needs the export format, which can be assumed to be TIFF here as well.

Reply:
Never mind, this is actually not needed. PIL understands the file name extension.